plone/app/layout/permissions.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('plone.app.layout')
+
+security.declarePublic('ShowToolbar')
```

Comment:
@gforcada I believe that just defines the module level attribute ``ShowToolbar`` defined below to be public.

But why is this ``permissions.py`` required at all? @agitator? I don't see it being imprted anywhere. And default roles for that permission are set in separate pulls for CMFPlone and plone.app.upgrade.